pages.fr/common/ghdl.md

```diff
@@ -0,0 +1,28 @@
+# ghdl
+
+> Simulateur à source ouverte pour le langage VHDL.
+> Plus d'informations: <http://ghdl.free.fr>.
+
+- Analyse un fichier de source VHDL et gérère un fichier objet:
```

Comment:
I didn't notice that the space before the `:` is missing.
```suggestion
- Analyse un fichier de source VHDL et gérère un fichier objet :
```
Could you please add the space in all the examples?

Reply:
Sure, just realized there is a spelling error too.